src/server/ranch/RanchDirector.cpp

```diff
@@ -530,9 +533,12 @@ void RanchDirector::HandleEnterRanch(
         for (const auto& housingRecord : *housingRecords)
         {
           housingRecord.Immutable([&response](const data::Housing& housing){
-            constexpr uint16_t IncubatorHousingId = 52;
+
             // Certain types of housing have durability instead of expiration time.
-            const bool hasDurability = (housing.housingId() == IncubatorHousingId) ;
+            const bool hasDurability = (housing.housingId() == SingleIncubatorId || housing.housingId() == DoubleIncubatorId);
+            if (hasDurability)
+              response.incubatorUseCount = housing.durability();
+              response.incubatorSlots = housing.incubatorSlots();
```

Comment:
```suggestion
            if (hasDurability) 
            {
              response.incubatorUseCount = housing.durability();
              response.incubatorSlots = housing.housingId() == DoubleIncubatorId ? 2 : 1;
            }
```

You can discard incubator slots from the data::Housing record